smartsim/_core/launcher/stepInfo.py

```diff
@@ -153,15 +153,16 @@ def __init__(
 
 
 class PBSStepInfo(StepInfo):  # cov-pbs
-
     @property
     def mapping(self) -> t.Dict[str, str]:
-        # see http://nusc.nsu.ru/wiki/lib/exe/fetch.php/doc/pbs/PBSReferenceGuide19.2.1.pdf#M11.9.90788.PBSHeading1.81.Job.States
+        # see http://nusc.nsu.ru/wiki/lib/exe/fetch.php/doc/pbs
+        # /PBSReferenceGuide19.2.1.pdf#M11.9.90788.PBSHeading1.81.Job.States
```

Comment:
Don't break the URL here. Just flag to ignore line length

Reply:
👍